ui/studio/views/sql/SqlView.tsx

```
@@ -875,119 +878,138 @@ export function SqlView(_props: ViewProps) {
+                hasUserEditedEditorValueRef.current = true;
+                latestEditorValueRef.current = value;
+                setEditorValue(value);
+              }}
+              placeholder="Write SQL..."
+              theme={isDarkMode ? "dark" : "light"}
+              value={editorValue}
+            />
+          </div>
+          {aiGenerationErrorMessage ? (
+            <div className="text-sm text-destructive">
+              <strong>AI SQL generation error:</strong> {aiGenerationErrorMessage}
```

ESLint

[error] Insert `{"·"}⏎·············`

(prettier/prettier)